### DISCUSSION TITLE
This plugin is malfunctioning and not working Midi2LR latest - LR10.4

### DISCUSSION THREAD

**Discussion by balgenbruder** [rsjaffe/MIDI2LR#597] [Problems]
Stuck
Purchased the controller - installed the software - donated - nothing.
Any help and ideas would be appreciated.

Best Bernd

**Comment by balgenbruder** [comment]
OK Version 4 shows but none of the assigments on the knobs does anything in LR

**Comment by balgenbruder** [comment]
Version 5 tells me malfuncioning I am running Catalina latest version

**Comment by balgenbruder** [comment]
Got it running with latest V4.

**Reply by orightok** [reply]
Hi, can you elaborate on how you got it finally running? I was on V3.XYZ till today and just installed the latest V4. It shows, it loads my older settings XML, and yet it does nothing in LR :-( Any ideas? Thanks and happy new year!!

Edit: NVM, it was't working while was connected to a USB splitter. Plugged it directly instead, and it works fine. Thanks anyway! :-)

**Reply by balgenbruder** [reply]
It was just the software version - V5 does not work for this MAC OS obviously.
Now thinking how to assign the functions to the device.

Best Bernd

**Comment by hdstead** [comment]
I am no expert, but this happened to me as well.  It turned out to be a
poor cable.  I purchased a quality one and it worked.  I did donate as once
you get this running you will not be wanting to live with the old sliders
and keyboard only.

Happy Christmas to you and yours and all one and all MIDI2LR users.
Hopefully, 2022 brings, with it, health and prosperity.

Doug Stead

On Wed, 22 Dec 2021 at 03:22, balgenbruder ***@***.***> wrote:

> Stuck
> Purchased the controller - installed the software - donated - nothing.
> Any help and ideas would be appreciated.
>
> Best Bernd
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rsjaffe/MIDI2LR/discussions/597>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL23NPH2MUSVHDMJ2GFNS6DUSGYIBANCNFSM5KSKMMLA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
                            ''~``
                           ( o o )
+---------------------.oooO--(_)--Oooo.--------------------------+
|
| Doug Stead, Chairman:
|
|                     Tri-M Technologies Inc.,
|               Tri-M Aviation Inc. &  Galt Air Inc.
|
|                            www.tri-m.com
|
|
|                       "Courage is resistance to fear,
|                                                        mastery of fear ,
|                                                  — not absence of fear."
|                           Mark Twain
|
|                         .oooO
|                          (  )     Oooo.
+---------------------------\ (--- (   )-------------------------+
                             \_)    ) /
                                   (_/

Perpetrators, collaborators, bystanders, victims: we can be clear about
three of these categories. The bystanders, however, act as a fulcrum and
"All that is necessary for the triumph of evil is that good people do
nothing".  If you are not a part of the solution, you are part of the
problem!  With apologies to Edmund Burke (1729 ~ 1797).

Confidentiality Notice: This e-mail and any files transmitted with it are
solely intended for the use of the recipient(s) and may contain information
that is confidential and privileged. If you receive this e-mail in error,
please advise me by return e-mail immediately. Please also disregard the
contents of the e-mail, deleting it and destroying all copies immediately.
